tests/provision_enterprise_server.sh

```diff
@@ -30,7 +30,8 @@ EOF
 install -o vagrant -g vagrant -m 0644 "$profile" ~vagrant/.bash_profile
 
 # setup module dependencies
-puppet module install puppetlabs/rabbitmq
+puppet module install puppet/rabbitmq
+puppet module install puppet/archive
```

Comment:
shouldnt archive get installed as a dependency of rabbitmq?

https://github.com/voxpupuli/puppet-rabbitmq/blob/master/metadata.json#L60